fault-remediation/pkg/initializer/init.go

```diff
@@ -64,6 +66,7 @@ func InitializeAll(ctx context.Context, params InitializationParams) (*Component
 		slog.Info("Log collector enabled")
 	}
 
+	//TODO: replace this with manager client
```

Comment:
AFAIK the manager client is cached, so we may want to think about the effects of the cache. Since we read the state of the remediation flow in our equivalence logic and we may end up operating on older information.